_data/docs.yml

```diff
@@ -46,6 +46,16 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  cugraph-ops:
+    name: cuGraph-Ops
+    path: cugraph-ops
+    desc: 'cuGraph-Ops aims to be a low-level, framework agnostic library providing commonly used computational operators for GNNs and other graph operations.'
+    ghlink: https://github.com/rapidsai/cugraph-ops
```

Comment:
cugraph-ops is closed source, hence I am not sure what to do about the link to GitHub. We could either link to cugraph or nothing at all.